src/steps/os/mod.rs

```diff
@@ -2,7 +2,7 @@
 pub mod dragonfly;
 #[cfg(target_os = "freebsd")]
 pub mod freebsd;
-#[cfg(target_os = "linux")]
+#[cfg(unix)]
```

Comment:
I rather that we don't do this. Instead, the `nix` step should use that module only in Linux

Reply:
Got it. Thanks!